.github/workflows/pr-ci.yml

```diff
@@ -43,8 +43,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-18.04
-          - ubuntu-20.04
+          - ubuntu-latest
```

Comment:
It's actually useful to test on all available ubuntu options, as it can detect different issues.  What versions are now usable?

Reply:
According to [github documentation](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources), ubuntu-20.04 (ubuntu-latest) and ubuntu-22.04.  I'm not sure why latest points to the older version.

Reply:
I'll update this one to point to the explicit versions - should coverity remain on latest?

Reply:
That doc does say `-latest` is the latest stable version, not necessarily the latest release. +1 to explicitly use the version numbers instead of latest.

Coverity is very picky, I was glad I got it working in the last go around, but just sneezing in that general direction might break it.  I do see a new release at https://scan.coverity.com/download/, so if the 2020.9 version we use doesn't work with ubuntu-20.04, you might want to try their release from this past August.

Reply:
thanks for the tips @rajachan, will take a look if we have issues.